### ISSUE TITLE
Banner subtitle overlaps banner

### ISSUE THREAD

**Issue by konsnos** [TheNAF/naflm#78]
Upgraded from a previous version of obblm and noticed this one.

![obblm_banner_subtitle](https://cloud.githubusercontent.com/assets/405674/12979193/428117de-d0de-11e5-945e-66ebfb152634.png)

**Comment by rythos42** [comment]
Perhaps someone with more history on the project can help me figure this one out -- I haven't seen this banner logo anywhere? And the colours aren't the same as current?

@konsnos Did you copy and replace the css folder when you did the upgrade? It looks like it might be solved by replacing your theme with the new one.

**Comment by Shteve0** [comment]
The stylesheet here is the default packaged with obblm. The subtitle I
introduced for the dynamic banner introduced in naflm.

Short term fix i think just track down that subtitle  and get rid (I think
funnyfingers moved it out of the htmlout or header file where I'd left it,
my memory's hazy). Long term maybe we need to call a banner file or include
in the general site settings - I'll look to get dynamic version of the
obblm default stylesheet in place too.

**Comment by thefloppy1** [comment]
Yeah this is the really old version of OBBLM stylesheet.

**Comment by konsnos** [comment]
Hey guys.

Thanks for the tips. I found the text at the file ./obblm/localsettings/settings_1.php so I changed it to an empty one and now I have no problem with it.

I didn't know that I can change the stylesheet/theme or that there are other ones. Can you point me to how I can change it?

PS: It took me some time to find out this version of obblm. I'm not sure if you are the correct persons to ask but could someone update the wiki http://nicholasmr.dk/obblmwiki/index.php/Main_Page to point to this version of the software?

**Comment by Shteve0** [comment]
In settings 1, there should be a stylesheet setting.  Change to 2 and
you'll get the NAF skin.

I believe thefloppy1's working on a new user guide but I'll certainly ask
Nicholas to add a note about the naflm project on his wiki!

**Comment by thefloppy1** [comment]
Yep I am working on the user manual right now, trying to make it as simple as possible.

**Comment by konsnos** [comment]
@Shteve0 Oddly enough that didn't work but instead the stylesheet option in settings_none.php did.

**Comment by thefloppy1** [comment]
So it looks like you have a mix of files from the original, and NAF Fork.
this $settings['banner_subtitle'] = 'New here?  Visit theNAF.net/Leagues for more information'; line is in the settings_1.php, and is from the NAF fork, the CSS files that you have are all from the original project.
Make a backup, then Copy all the CSS folders from the NAF fork over the servers current CSS files.

**Comment by Shteve0** [comment]
I think actually it might just be that there's no League ID=1 so it doesn't
call that settings file, it calls the _none?

**Comment by konsnos** [comment]
Maybe since I haven't yet imported the data. To be honest I'm happy with the state it is now so I'm not going to fiddle with it anymore until I import the data of our plays.

Just to add some info to this issue that I've discovered through my search, I've noticed that the NAF theme in the css was the one marked 1, while the bloody in the photo of the issue is marked 3. So I'm not sure how the one I had was the default, but that would make sense that I can't set it back. I tried logging out and setting the stylesheet option to 3 and I've got the bloody theme again. However in file index.php the file checks my profile options for the theme. So after that I've checked out the user profile options, and I only had the option for the 1 the Classic. I'm not sure why I wasn't presented with the 2nd and 3rd stylesheet but I didn't search any further. I hope this becomes useful if someone tries to figure this out.
